app/src/main/java/com/aptoide/uploader/apps/view/MyStoreFragment.java

```diff
@@ -131,16 +182,113 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         });
   }
 
+  @Override public void toggleSubmitButton(boolean status) {
+
+    if (status) {
+      TranslateAnimation translateAnimation =
+          new TranslateAnimation(0, 0, 0, -storeBanner.getHeight());
+      translateAnimation.setDuration(200);
+      translateAnimation.setFillAfter(true);
+
+      translateAnimation.setAnimationListener(new Animation.AnimationListener() {
+        @Override public void onAnimationStart(Animation animation) {
+        }
+
+        @Override public void onAnimationEnd(Animation animation) {
+          mainScreen.setTranslationY(-storeBanner.getHeight());
+          mainScreen.clearAnimation();
+        }
+
+        @Override public void onAnimationRepeat(Animation animation) {
+
+        }
+      });
+
+      mainScreen.startAnimation(translateAnimation);
+      submitButton.startAnimation(slideBottomUp);
+    }
+    else {
+      TranslateAnimation translateAnimation =
+          new TranslateAnimation(0, 0, 0, storeBanner.getHeight());
+      translateAnimation.setDuration(200);
+      translateAnimation.setFillAfter(true);
+
+      translateAnimation.setAnimationListener(new Animation.AnimationListener() {
+        @Override public void onAnimationStart(Animation animation) {
+        }
+
+        @Override public void onAnimationEnd(Animation animation) {
+          mainScreen.setTranslationY(0);
+          mainScreen.clearAnimation();
+        }
+
+        @Override public void onAnimationRepeat(Animation animation) {
+
+        }
+      });
+
+      mainScreen.startAnimation(translateAnimation);
+      submitButton.startAnimation(slideBottomDown);
+    }
+  }
+
+  public void setUpSubmitButtonAnimation() {
+
+    final Animation.AnimationListener showBottom = new Animation.AnimationListener() {
+      @Override public void onAnimationStart(Animation animation) {
+        submitButton.setVisibility(View.VISIBLE);
+      }
+
+      @Override public void onAnimationEnd(Animation animation) {
+
+      }
+
+      @Override public void onAnimationRepeat(Animation animation) {
+
+      }
+    };
+
+    final Animation.AnimationListener hideBottom = new Animation.AnimationListener() {
+      @Override public void onAnimationStart(Animation animation) {
+        submitButton.setVisibility(View.GONE);
+      }
+
+      @Override public void onAnimationEnd(Animation animation) {
+
+      }
+
+      @Override public void onAnimationRepeat(Animation animation) {
+
+      }
+    };
+
+    slideBottomDown = AnimationUtils.loadAnimation(getContext(), R.anim.slide_bottom_down);
+    slideBottomUp = AnimationUtils.loadAnimation(getContext(), R.anim.slide_bottom_up);
+    slideBottomUp.setAnimationListener(showBottom);
+    slideBottomDown.setAnimationListener(hideBottom);
+  }
+
+  public Observable<Boolean> handleTitleChange(boolean status) {
```

Comment:
hardcoded strings, this will break translations.

Reply:
Fixed